src/actions/detailGif.js

```diff
@@ -0,0 +1,30 @@
+const GET_SINGLE_GIF_REQUESTED = 'GET_SINGLE_GIF_REQUESTED';
+const GET_SINGLE_GIF_COMPLETED = 'GET_SINGLE_GIF_COMPLETED';
+const GET_SINGLE_GIF_FAILED = 'GET_SINGLE_GIF_FAILED';
+
+const getSingleGifRequested = gifId => ({
+  type: GET_SINGLE_GIF_REQUESTED,
+  payload: gifId,
```

Comment:
Action creators should wrap the payload inside an object:

```
const getSingleGifRequested = (gif) => (
  type: 'GET_SINGLE_GIF_REQUESTED',
  payload: { gifId },
);
```
This is in order to have a standard way of transporting payloads, and
comply with the Flux standard.